README.md

```diff
@@ -1,2 +1,216 @@
+<img src="/images/ERD.jpeg" alt="Logo" width="800" height="500">
+  </a>
+
+## Built With:
+This project was built using these technologies:
+* Ruby
+* Ruby on Rails
+* React
+* Redux
+* HTML
+* CSS
+* JavaScript
+* PostgreSQL, pgAdmin 4
+* JSON Web Token (JWT)
+* API integration
+* Adobe Illustrator
+* Adobe After Effect
+* Rubocop
+* Capybara
+* Rspec
+* OpenApi(Swagger)
+* GitHub, Gitflow
+* Visual Studio
+
+
+## Usage:
+
+To have this app on your pc, you need to:
+
+  - Clone with SSH:
+  ```
+git@github.com:Home-Sharing-Team/final-capstone-Project-Back-End.git
+  ```
+  - Clone with HTTPS
+  ```
+https://github.com/Home-Sharing-Team/final-capstone-Project-Back-End.git
+  ```
+
+* In the project directory, you can run:
+
+  - `$ bundle install` - installs all the dependencies required by the project
+
+  Run 'Rails s' command to run the Ruby on Rails server and launch your browser to see the project
+
+## Testing:
+
+Tu run the test in your terminal, please type: `rspec spec/capybara`
+
+
+## Kanban Board:
+
+Link to the project Kanban board: <a href="https://github.com/orgs/Home-Sharing-Team/projects/2">Click here</a>.
+
+Link to the screenshot of the initial state of the Kanban board: <a href="https://user-images.githubusercontent.com/64566209/199840132-dde39cfe-be8a-4172-9bf5-f5d0485ce13f.png">Click here</a>.
+
+
+
+## Authors:
+
+👤 **Ulises Largaespada**
+
+- GitHub: [@ulisesedu7](https://github.com/ulisesedu7)
+
+- Twitter: [@LongswordMusic](https://twitter.com/LongswordMusic)
+
+- LinkedIn: [Ulises Largaespada](https://www.linkedin.com/in/ulises-largaespada-45570b1a4/)
+
+
+👤 **Gabriel Santo**
+
+- GitHub: [@gvgesanto2](https://github.com/gvgesanto2)
+
+- Twitter: [@GabrielSanto997](https://twitter.com/GabrielSanto997)
+
+- LinkedIn: [Gabriel Santo](https://linkedin.com/in/gabriel-santo-5882a71b2/)
+
+
+👤 **David Verde Alvarez**
+
+- This is My **GitHub**: [![@David-Verde](https://img.shields.io/github/followers/omarramoun?label=David&style=social)](https://github.com/David-Verde)
+
+- This is My **Twitter**: [![@Unyielding1](https://img.shields.io/twitter/follow/omarramoun?label=David16&style=social)](https://twitter.com/UnyieldingOne)
+
+
+👤 Dario Alessio
+
+- This is My **GitHub**: [![@DarioAlessioR](https://img.shields.io/github/followers/omarramoun?label=Dario&style=social)](https://github.com/DarioAlessioR)
+
+- This is My **Slack**: https://microverse-students.slack.com/team/U039GCFRK9B
+
+- This is My **LinkedIn**: https://www.linkedin.com/in/dario-alessio-3a3b7911b
+
+
+## Show your support:
+
+Give a :star: if you like this project!
+
+
+## Acknowledgements:
+
+- Based on the [design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign) by [Murat Korkmaz](https://www.behance.net/muratk) on Behance.
+- Hat tip to anyone whose code was used
+- Microverse
+- Our supportive families 😊
+
+## License:
+This project is [MIT](https://github.com/David-Verde/final-capstone-Project-Back-End/blob/develop/LICENSE) licensed.
```

Comment:
- [ ]  This link to the LICENSE file does not work, because it does not point to the correct URL.  Kindly correct it.

Reply:
Thank you @Meltrust . We already fixed the license link.